src/services/utilities.ts

```diff
@@ -469,6 +469,38 @@ namespace ts {
         }
     }
 
+    /**
+     * Get the corresponding JSDocTag node if the position is in a jsDoc commet
+     */
+    export function getJsDocTagAtPosition(sourceFile: SourceFile, position: number): JSDocTag {
+        let node = ts.getTokenAtPosition(sourceFile, position);
+        let jsDocComment: JSDocComment;
+        if (node.jsDocComment) {

```

Comment:
There are actually two cases:
1. the current node is ``var``, ``let`` or ``const``. Then the parent level is ``VariableDeclarationList``, and the grandparent level is ``VariableStatement``, which is what we want. So for those three types of tokens we need to go up 2 layers in the tree
2. for others go up one level is fine.